src/components/MarkerInfo/BoxInfo/BoxInfo.module.css

```diff
@@ -88,6 +88,12 @@
   width: 30px;
 }
 
+.desktop-delete-box-button {
+  position: fixed !important;
+  right: 10px;
```

Comment:
Bit of a nitpick but the delete box is just slightly off

![image](https://user-images.githubusercontent.com/70079773/168454445-ffd283e1-e929-4728-bfcd-ad44d302b13f.png)

Reply:
beautiful